cogment_verse/model.py

```diff
@@ -52,3 +54,28 @@ def deserialize_model(cls, serialized_model):
         Returns:
             model: the deserialized model
         """
+
+    @classmethod
+    async def retrieve_model(cls, actor_session, model_id, iteration) -> Model:
```

Comment:
It seems better to pass the model_registry here to keep the dependency chain more contained.

```suggestion
    async def retrieve_model(cls, model_registry, model_id, iteration) -> Model:
```

Reply:
Sure